### DISCUSSION TITLE
Is Ferdium Syncing Service E2EE

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferdium/ferdium-app#1698] [Q&A]
Is the Ferdium Syncing service E2EE were the Ferdium server devs cannot even read your data and metadata? Or can Ferdium read your data or some of the data?

**Comment by vraravam** [comment]
For the hosted servers, only the following items are synced:
1. list (and order) of services that are configured in your installation
2. list (and order) of workspaces
3. other preferences that you would set via the Preferences UI are synced.

We do NOT sync the session information since that could result in security issues.

**Reply by trymeouteh** [reply]
Is any of this data encrypted using E2EE?

**Comment by SpecialAro** [comment]
> Ferdium Syncing service

What do you mean by "Ferdium Syncing service" ? Ferdium Server?

**Reply by SpecialAro** [reply]
Lets focus this convo on: https://github.com/ferdium/ferdium-app/issues/1709. I'll close this discussion.
